crates/gateway-types/src/reply.rs

```diff
@@ -17,8 +17,11 @@ pub struct Block {
     pub block_number: BlockNumber,
     /// Excluded in blocks prior to Starknet 0.9
     #[serde_as(as = "Option<GasPriceAsHexStr>")]
-    #[serde(default)]
-    pub gas_price: Option<GasPrice>,
+    #[serde(default, alias = "gas_price")]
+    pub eth_l1_gas_price: Option<GasPrice>,
+    #[serde_as(as = "Option<GasPriceAsHexStr>")]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub strk_l1_gas_price: Option<GasPrice>,
```

Comment:
I assume they've kept the original field names in historic blocks?

Reply:
Maybe add a note to remove the options once v0.13 is deployed everywhere.

Reply:
I've added a TODO.